src/h5web/providers/jupyter/utils.ts

```diff
@@ -110,19 +100,8 @@ export function convertDtype(dtype: string): HDF5Type {
       };
 
     case 'S':
-      return {
-        class: HDF5TypeClass.String,
-        charSet: 'ASCII',
-        length: length || 'H5T_VARIABLE',
-      };
-
-    case 'U':
```

Comment:
h5py does not support unicode `U` strings: https://docs.h5py.org/en/stable/faq.html#what-datatypes-are-supported